addon/model.js

```diff
@@ -391,6 +391,11 @@ export default class MegamorphicModel extends Ember.Object {
 
   // TODO: drop change events for unretrieved properties
   setUnknownProperty(key, value) {
+    if (key === OWNER_KEY) {
+      // 2.12 support; later versions avoid this call entirely
```

Comment:
@stefanpenner nope and it won't show up as part of the PR after #19 lands.  If #19 changes i'll rebase this to drop that commit.

Ah i'll just do so anyway but it means travis will for sure fail on lts 2.12